packages/spaces/src/lib/SpacesImage.tsx

```diff
@@ -0,0 +1,72 @@
+import { CircularProgress } from '@availity/mui-progress';
+import { Img } from 'react-image';
+import { useSpaces, useSpacesContext } from './Spaces';
+
+type BaseSpacesImageProps = {
+  imageType?:
+    | 'url'
+    | 'images.logo'
+    | 'images.tile'
+    | 'images.billboard'
+    | 'images.promotional'
+    | 'images.promotionalHover';
+  fallback?: string;
+  id?: string;
+};
+
+interface SpacesImageSpaceId extends BaseSpacesImageProps {
```

Comment:
I think we should use a union instead of extending here. eg: 
```ts
type SpacesImageSpaceId = {
  // new types
} & BaseSpacesImageProps
```

Reply:
fixed